system/Database/Postgre/Connection.php

```diff
@@ -325,35 +325,55 @@ protected function _indexData(string $table): array
      */
     protected function _foreignKeyData(string $table): array
     {
```

Comment:
added delete and update rules and match option. Fixed improper join using ordinal_position.